### DISCUSSION TITLE
Error divergence during impact wall simulation

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1578] [Q&A] [answered]
Hi, I'm trying to run the following simulation which have a rigid part impact another one following a specific load curve. The simulation for some reason diverge and I cannot understand the reason. Does anyone have any suggestion? Is an error of conversion of the file from LS-DYNA to OpenRadioss?
[high_def.zip](https://github.com/OpenRadioss/OpenRadioss/files/11744542/high_def.zip)

**Comment by PaulAltair** [comment, marked as answer]
The model seems to run ok, but will fail somewhere around 12.5ms, because your loadcase is trying to push the plate further than the can is long...? Your can is 200mm long and the job fails when it is crushed by around 186mm, but the request for motion goes up to 300mm, and later to 500mm, which is not physically possible for a 200mm long can, so I guess there is a mistake in you set up here?

The model is running for me up to that point, but there are some things I would change, 
1 you have tied nodes to a rigid mat, it would be better to add those nodes as constrained extra nodes. 
2 your densities seem to be wrong by a factor of 1000 (if units are kg mm ms) 
3 you could afford to add some mass to decrease the run time. (I scaled timestep here to 2.7e-4)
I've attached my modified version
[Density_Correction_OR_TS_Scale.zip](https://github.com/OpenRadioss/OpenRadioss/files/11746936/Density_Correction_OR_TS_Scale.zip)

**Reply by FGarelli** [reply]
Yes, you are right, the load is not set well.
Thank you for the corrections and for the model, it helps me a lot!!

**Reply by FGarelli** [reply]
Hi Paul, the simulation doesn't follow the direction that i imposed during the prescribed motion constrain, do you have an idea of why? How should I set the prescribed motion to go just along one dof?
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/9bd5ea24-f37a-4e3f-bfab-e0720f46b74a)

**Reply by PaulAltair** [reply]
You are getting the motion requested in X (at c of g of the platen) but the prescribed motion affects only the DOF on which it is applied (X), the others remain free. If you also want to constrain the platen from moving in Y,Z and from rotating you can add constraints on the *MAT_RIGID using CMO flag and CON1, CON2: here set 4 and 7 (Y,Z disp and X,Y,Z rot), see [MAT_RIGID in help](https://help.altair.com/hwsolvers/rad/topics/solvers/rad/mat_020_rigid_lsdyna_r.htm)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/2fe558b9-9cfe-4890-bca7-1453bf1acb4a)

**Reply by FGarelli** [reply]
Thank you!
